src/checkout/CheckoutPage.tsx

```diff
@@ -4,13 +4,16 @@ import React, {FC} from 'react'
 import CheckoutForm from 'src/checkout/CheckoutForm'
 import SuccessOverlay from 'src/checkout/SuccessOverlay'
 import CheckoutProvider from 'src/checkout/context/checkout'
+import {isFlagEnabled} from 'src/shared/utils/featureFlag'
+import ZuoraOutagePage from 'src/shared/components/zuora/ZuoraOutagePage'
 
 const CheckoutV2: FC = () => {
   return (
     <CheckoutProvider>
       <>
         <SuccessOverlay />
-        <CheckoutForm />
+        {isFlagEnabled('quartzZuoraDisabled') && <ZuoraOutagePage />}
+        {!isFlagEnabled('quartzZuoraDisabled') && <CheckoutForm />}
```

Comment:
this could probably be converted into a ternary right? I think that would read better